lib/tx.js

```diff
@@ -56,15 +56,34 @@ function postToTransaction(sdk, url, options) {
 }
 
 function getPollFn(sdk, res, ref) {
-  return function (delay) {
-    if (!delay && delay !== 0) {
+  return function (options) {
+    var delay;
+    var rememberDevice;
+
+    if (util.isNumber(options)) {
+      delay = options;
+
+    } else if (util.isObject(options)) {
+      delay = options.delay;
+      rememberDevice = options.rememberDevice;
+    }
+
+    if (!delay) {
       delay = DEFAULT_POLLING_DELAY;
     }
 
     // Get the poll function
     var pollLink = util.getLink(res, 'next', 'poll');
     function pollFn() {
-      return http.post(sdk, pollLink.href, getStateToken(res), true, true);
+      var href = pollLink.href;
+      if (rememberDevice !== undefined) {
+        if (rememberDevice) {
+          href += '?rememberDevice=true';
+        } else {
+          href += '?rememberDevice=false';
+        }
+      }
```

Comment:
I think if you passed `rememberDevice: false` as a dev, and didn't see that appended to your request, it'd be confusing. So I'm down to simplify it (verify defaults to rememberDevice=false anyway) if it's not too helpful.